package/python3-img-proof.spec

```diff
@@ -1,5 +1,5 @@
 #
-# spec file for package python3-img-proof
+# spec file for package python-img-proof
```

Comment:
The file itself also needs renaming python3-img-proof.spec -> python-img-proof.spec